packages/apps/stock/src/screens/supplierArrivals/SupplierArrivalLineDetailScreen.js

```diff
@@ -234,6 +244,15 @@ const SupplierArrivalLineDetailScreen = ({route, navigation}) => {
           }
           isScrollViewContainer={true}
         />
+        <TouchableOpacity
+          style={styles.trackingNumberContainer}
+          onPress={handleAddTrackingNumber}
+          disabled={readonly}>
+          <Text writingType="important" style={styles.trackingNumberText}>
+            {I18n.t('Stock_AddTrackingNumber')}
+          </Text>
+          <Icon name="plus-lg" size={24} style={styles.action} />
+        </TouchableOpacity>
```

Comment:
Create component for that
Also, it should be display only on conditions : product.trackingNumberConfiguration != null and trackingNumber == null
User should be also able to select an existing tracking number